.ci/release.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+ROOT_DIR=$(cd "$(dirname "$0")"; pwd)/..
+
+docker login -u ${DOCKER_HUB_LOGIN} -p ${DOCKER_HUB_PASSWORD}
+
+# Run tests and all the other checks on repo
+docker build -f ${ROOT_DIR}/build/CI/Dockerfile ${ROOT_DIR}
+
+# create and push new tag
+git tag $GITHUB_TAG
+git push origin $GITHUB_TAG
+
+# checkout to new tag
+git checkout $GITHUB_TAG
+
+# Unified Plugin Broker
+docker build -t eclipse/che-unified-plugin-broker:${IMAGE_TAG} -f ${ROOT_DIR}/build/unified/Dockerfile ${ROOT_DIR}
```

Comment:
I would add checks for `IMAGE_TAG` and `GITHUB_TAG ` and fail if these are not set, this will protect this script from being executed locally with not ENVs set

Reply:
Done